renovate.json5

```diff
@@ -0,0 +1,78 @@
+{
+	extends: [ 'config:recommended', 'default:pinDigestsDisabled' ],
+	packageRules: [
+		{
+			extends: [ 'monorepo:wordpress', 'schedule:daily' ],
+			separateMajorMinor: false,
+			prPriority: 3,
+		},
+		{
+			extends: [ 'group:nodeJs' ],
+			prPriority: 2,
+		},
+		{
+			extends: [ 'monorepo:react', ':widenPeerDependencies' ],
+			prPriority: 2,
+			ignorePaths: [ 'packages/interpolate-components' ],
+		},
+		{
+			groupName: 'redux-related packages',
+			matchPackagePatterns: [ 'redux' ],
+			matchPackageNames: [ 'react-redux' ],
+			prPriority: 2,
+		},
+		{
+			groupName: 'typescript-related packages',
+			matchPackagePatterns: [ 'typescript-eslint' ],
+			matchPackageNames: [ 'typescript' ],
+			prPriority: 2,
```

Comment:
**Question (non-blocking):** Mostly out of curiosity, why don't we want `@types` packages included in the renovate update PRs? Have we had issues with them getting too far ahead?

Reply:
We do want `@types` to update! It just doesn't make sense to update them with Typescript -- they should be updated when the corresponding package is updated. For example, `@types/react` probably only needs to be updated when `react` is updated.

By the way, the config automatically updates everything by default when we don't specify a certain package (unless we manually ignore it). The groups we set up just add our own own settings on top of that, to help us be more organized.